src/checks/bannedFunctions.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+
+/**
+ * @description disallows use of a specific list of key words
+ * @param {string} [line] curr line being linted
+ * @return {boolean} true if translate3d is used, false if not
+ */
+var bannedFunctions = function( line ) {
+	var found = false
+	var bannedFunctionList = this.config['bannedFunctions'] || []
+	var index = -1
+
+	bannedFunctionList.forEach( function( func ) {
+		index = line.indexOf( func )
+		if ( index !== -1 ) {
+			found = true
+			this.msg( func + ' is currently banned', index )
```

Comment:
`is currently banned` seems like a weird way to formulate it, IMO.

What about copying eslint's approach? So something like `unexpected function 'function-name' encountered`